helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -467,19 +475,13 @@ public void resetMaxTopStateHandoffGauge() {
         _numPendingLoadRebalanceReplicas,
         _numRecoveryRebalanceThrottledReplicas,
         _numLoadRebalanceThrottledReplicas,
-        _externalViewIdealStateDiff,
-        _successfulTopStateHandoffDurationCounter,
-        _successTopStateHandoffCounter,
-        _failedTopStateHandoffCounter,
-        _maxSinglePartitionTopStateHandoffDuration,
-        _partitionTopStateHandoffDurationGauge,
+        _externalViewIdealStateDiff, _successfulTopStateHandoffDurationCounter,
+        _successTopStateHandoffCounter, _failedTopStateHandoffCounter,
+        _maxSinglePartitionTopStateHandoffDuration, _partitionTopStateHandoffDurationGauge,
         _partitionTopStateHandoffHelixLatencyGauge,
-        _partitionTopStateNonGracefulHandoffDurationGauge,
-        _totalMessageReceived,
-        _totalMessageReceivedCounter,
-        _numPendingStateTransitions,
-        _rebalanceState
-    );
+        _partitionTopStateNonGracefulHandoffDurationGauge, _totalMessageReceived,
+        _totalMessageReceivedCounter, _numPendingStateTransitions, _rebalanceState,
```

Comment:
earlier style was keeping metrics per-line, you are combining 2 per-line. any particular reason?

Reply:
Auto formatting did that. Let me change it back.